nautobot/extras/views.py

```diff
@@ -455,6 +457,25 @@ class CustomFieldBulkDeleteView(generic.BulkDeleteView):
     queryset = CustomField.objects.all()
     table = tables.CustomFieldTable
 
+    def construct_custom_field_delete_tasks(self, queryset):
+        """
+        Helper method to construct a list of celery tasks to execute when bulk deleting custom fields.
+        """
+        tasks = [
+            delete_custom_field_data.si(obj.name, set(obj.content_types.values_list("pk", flat=True)))
+            for obj in queryset
+        ]
+        return tasks
+
+    def perform_pre_delete(self, request, queryset):
+        if not get_worker_count(request):
+            messages.error(request, "Unable to run job: Celery worker process not running.")
```

Comment:
Can we make this message a bit less generic? Perhaps:

```suggestion
            messages.error(
                request,
                "Celery worker process not running. Object custom fields may fail to reflect this deletion."
            )
```